### ISSUE TITLE
Nothing alike

### ISSUE THREAD

**Issue by chrisbward** [myshell-ai/OpenVoice#43]
Not sure what's happening here - I managed to spin this up in the local gradio app, recorded my own voice, but inference gave me an american-sounding output - I'm British - is that expected?

Thanks!

**Comment by chrisbward** [comment]
I tried mimicking different voices on my microphone, but the output is always the same - is the app broken?

**Comment by chrisbward** [comment]
Okay, so a little further investigation;

I have copied demo_speaker1.mp3 to someother.mp3 and dragged this in to Gradio - and the voice is not cloned.

If I drag demo_speaker1.mp3 in, it works fine - so I do not think there is any inference happening at all. It's determined by filename.

**Comment by pixelass** [comment]
I tried this in collab and did several recordings. 
Result: It is not good... at all, I mean it is VERY bad (in terms of likeness). Sorry, but it seems to be very biased to certain voices that it can copy.

Just another overhyped no-good model :(

**Comment by chrisbward** [comment]
I think there are bugs in the gradio app and the cloning is not attempted at all.

**Comment by chrisbward** [comment]
You'll notice if you switch the files around in the samples folder, it uses the original samples voice, even when removed

**Comment by basher0** [comment]
Why the voice in the paper samples sounds perfect, but when I run it locally, it doesn't sound anything like it. StyleTTS 2 is much better.

**Comment by pixelass** [comment]
> Why the voice in the paper samples sounds perfect, but when I run it locally, it doesn't sound anything like it. StyleTTS 2 is much better.

Because good examples gives many stars and hopefully a lot of money. 

Fake it till you make it. duh :)

**Comment by iwoolf** [comment]
I installed it locally and tried my own voice and got a standard voice instead. I tried the supplied demos, and they all came out exactly the same standard voice I can't figure how to change the accent, there are no example prompts.

**Comment by Zengyi-Qin** [comment]
Hi All - Regarding the accent, please read the paper carefully before judging the results. The accent should be controlled by the base speaker model. The **tone color converter does not clone your accent.** This demo only provides control over emotion, and the accent is default to American accent. The users can use their own base speaker model (English accent) to replace the base speaker model in OpenVoice. The OpenVoice framework provides sufficient flexibility to do it.

**Comment by Tpann2518** [comment]
ok

**Comment by davechilds** [comment]
I have also tried the app as a local install and the voices come back sounding nothing like the reference audio. They all seem to sound like a teenage boy, with slight differences when different references are used. I tried resampling the original audio, in case they needed to be a specific KBit or Hz value, but this did not make any difference. Is there any reason why the results are so different from what I expected from the examples given on the website?

**Comment by Zengyi-Qin** [comment]
@hehuan2363 could you try the lepton demo and see if anything changed https://www.lepton.ai/playground/openvoice

**Comment by YKefasu** [comment]
> @hehuan2363 could you try the lepton demo and see if anything changed https://www.lepton.ai/playground/openvoice

It have same result

**Comment by pixelass** [comment]
> @hehuan2363 could you try the lepton demo and see if anything changed https://www.lepton.ai/playground/openvoice

I also tested this. Same inaccuracy on several voices I've tried. None of them come close to the examples.

**Comment by Aegon95** [comment]
> @hehuan2363 could you try the lepton demo and see if anything changed https://www.lepton.ai/playground/openvoice

Same result, The output is no where close and sounds childish

**Comment by DKRacingFan** [comment]
I'm sorry but this model is very inaccurate and bad. Hopefully you update it to significantly improve it as all the voices I tried are bad. Joe Biden, MattVidProAi, MrBeast etc...

**Comment by RASPIAUDIO** [comment]
I agree the result is very disapointing it almost feels like it is a scam. There are other older opensource models that do it better:
https://github.com/coqui-ai/TTS

**Comment by Zengyi-Qin** [comment]
https://github.com/myshell-ai/OpenVoice/blob/main/QA.md

**Comment by RASPIAUDIO** [comment]
> You can find the answers here https://github.com/myshell-ai/OpenVoice/blob/main/QA.md

Evrything seems OK, but I have this warning; not sure if it is a problem ?


(openvoice) PS C:\holobot\prod\OpenVoice> & C:/Users/olivi/anaconda3/envs/openvoice/python.exe c:/holobot/prod/OpenVoice/testopenvoice2.py
Loaded checkpoint 'checkpoints/converter/checkpoint.pth'
missing/unexpected keys: [] []
[(0.0, 13.33), (13.358, 23.666), (23.886, 38.002), (38.67, 51.144)]
after vad: dur = 50.228
C:\Users\olivi\anaconda3\envs\openvoice\lib\site-packages\wavmark\models\my_model.py:25: UserWarning: istft will require a complex-valued input tensor in a future PyTorch release. Matching the output from stft with return_complex=True.  (Triggered internally at C:\cb\pytorch_1000000000000\work\aten\src\ATen\native\SpectralOps.cpp:980.)
  return torch.istft(signal_wmd_fft, n_fft=self.n_fft, hop_length=self.hop_length, window=window,

**Comment by jackyin68** [comment]
Oh, No。But how to solve this. May give a example in more detail!!

**Comment by aytimothy** [comment]
I managed to get everything running on Ubuntu running in WSL on Windows 10. The VM is using Python 3.9.13. However...

Is it just me, or is the website demo setup with different models or weights? Or maybe I haven't set it up properly...
But, on the website, it is more accurate to the voice than my local demo.

Even with the demo voices provided, both v1 and v2 are less accurate than the website when run on my machine.
